Tests/Code/CodeTests.premake5.lua

```diff
@@ -0,0 +1,25 @@
+project "CodeTests"
+    kind "ConsoleApp"
+    dependson { "Catch2", "Engine" }
+
+	targetdir "%{wks.location}/Build/Bin/Tests/%{prj.name}"
+	objdir "%{wks.location}/Build/Obj/Tests/%{prj.name}"
+
+    defines { "_TEST" }
+
+    files { "**.cpp" }
+
+    includedirs {
+        "%{prj.location}/../../Vendor/Catch2/single_include/",
+        "%{prj.location}/../../Code/Game/",
+    }
+
+    vpaths {
+        ["Headers"] = { "**.h", "**.hpp" },
+        ["Sources/*"] = { "**.c", "**.cpp" },
+        ["Documentation"] = { "**.md", "**.txt" },
```

Comment:
Although I doubt this will matter - I changed the `Engine` and `Game` premake files to 'Docs' to be a bit shorter cause I realized 'Documentation' almost gets cut off in VS 😂